conf/proxy.conf

```diff
@@ -18,10 +18,10 @@
 #
 
 # Zookeeper quorum connection string (comma-separated)
-zookeeperServers=
+zookeeperServers=localhost:2181
```

Comment:
Leave empty so that we get an error if not filled out instead of having the zk client trying and retrying to connect